docs/project-configuration/subgraphs/working-with-multiple-repositories.mdx

```diff
@@ -0,0 +1,163 @@
+---
+sidebar_position: 3
+sidebar_label: Split Subgraphs across repositories
+description: "Learn how to split subgraphs across multiple repositories."
+keywords:
+  - hasura
+  - hasura ddn
+  - subgraphs
+  - multiple subgraphs
+  - parent repo
+  - multiple repositories
+  - mutli-repo
+  - ddn advanced
+  - advanced
+  - CLI
+  - context
+---
+
+# How to Split Subgraphs across Repositories
+
+## Introduction
+
+You can split your subgraphs into multiple repositories while unifying them under a single "parent" project.
+
+:::warning DDN Advanced Plan required
+
+In order to utilize multi-repository collaboration, you must have an active
+[DDN Advanced Plan](https://hasura.io/pricing).
+
+:::
+
+## Create the Initial Project
+
+Begin by creating a "parent" project that will serve as the coordinated supergraph for your independent subgraph
+repositories. This central project will manage provisioning subgraphs, inviting collaborators, and maintaining the
+overall API.
+
+### Step 1. Initialize a New Local Project
```

Comment:
Under "tutorials", the steps titles are not capitalized, while here they are. Can we make this consistent?